WalletWasabi/Crypto/ZeroKnowledge/ProofSystem.cs

```diff
@@ -11,7 +12,7 @@ public static class ProofSystem
 	{
 		private static GroupElement O = GroupElement.Infinity;
 
```

Comment:
@kiminuo suggested:

```suggestion

		private readonly static EmptyLabel = new byte[0];

```

(untested, i added `static`)

Reply:
No, this is not what we have to do. Currently the `Transcript` has a hole for the label and we need to fill it with the real label. It is something that can wait and that's why we are deferring the decision but it cannot be an empty byte array.